### ISSUE TITLE
google.com: breakage

### ISSUE THREAD

**Issue by Ryan147** [uBlockOrigin/uAssets#25779]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

When Ublock is on I see several entries in Google SERP not being shown. When I turn it off the entries are shown.

See attachments for the example.

![screencapture-google-search-2024-10-24-17_22_44](https://github.com/user-attachments/assets/eefb30b2-13fe-45fc-8b40-c2aa1f6f75f1)
![screencapture-google-search-2024-10-24-17_23_09](https://github.com/user-attachments/assets/9729852e-f5fa-469b-b4a9-e1d646c98b3b)

### A specific URL where the issue occurs.

```yaml
Google search
```

### Steps to Reproduce

1. Have uBO turned on
2. Go to Google Search and do a search. 

### Expected behavior

All SERP entries being shown

### Actual behavior

Some are hidden

### uBO version

1.60

### Browser name and version

Chrome latest

### Operating System and version

Windows 10

**Comment by stephenhawk8054** [comment]
Test again in a new Chrome profile with only uBO and its default settings.

**Comment by gorhill** [comment]
Go to the webpage with the issue > Click the [uBO toolbar icon](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface) > Click the [chat icon 💬](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Click "Troubleshooting Information" > Report content here (https://www.reddit.com/link/173jmog/video/bogwf92af9tb1/player)

**Comment by Ryan147** [comment]
Here it is:

uBlock Origin: 1.60.0
Chromium: 130
filterset (summary):
 network: 143954
 cosmetic: 54473
 scriptlet: 26859
 html: 0
listset (total-discarded, last-updated):
 added:
  https://raw.githubusercontent.com/laylavish/uBlockOrigin-HUGE-AI-Blocklist/main/list.txt: 4232-36, 8h.15m
  ublock-annoyances: 5969-32, 15m Δ
 default:
  user-filters: 0-0, never
  easylist: 80260-47, 15m Δ
  easyprivacy: 53189-21, 15m Δ
  plowe-0: 3545-999, 1d.23h.3m
  ublock-badware: 11576-24, 15m Δ
  ublock-filters: 40400-358, 15m Δ
  ublock-privacy: 1368-26, 15m Δ
  ublock-quick-fixes: 187-13, 15m Δ
  ublock-unbreak: 2491-33, 15m Δ
  urlhaus-1: 23800-0, 17h.15m
filterset (user): [empty]
trustedset:
 added: [array of 86 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2889 ms
 maxAssetCacheWait: 2032 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 5
 network:
  github.com: 3
  sentry.io: 2


Subsequently I found out why the Google SERP was partly "corrupt." It was because of the AI blocklist that the list above shows. When I turned it off the issue did not appear anymore.

Thanks for this gorhill!
